apps/src/templates/manageStudents/Table/UsStateColumn/BulkSetModal/style.scss

```diff
@@ -0,0 +1,44 @@
+@import 'color';
+@import 'font';
+
+#us-state-column-bulk-set-modal {
+  font-weight: $medium-font-weight;
+
+  & a {
+    color: $default_text;
+    text-decoration: underline;
+  }
+
+  & .title {
+    margin: 20px 0;
+  }
+
+  & hr {
+    margin: 0 20px !important;
```

Comment:
Generally `!important` is discouraged, are there other ways of increasing specificity here?

Reply:
The issue is that I need to rewrite the styles defined in the tag; otherwise, I would have to pass them through props across several layers of components:
```html
<hr style="margin: 0px; border-color: rgb(198, 202, 205);">
```

Reply:
Well that's unfortunate that this uses an inline style :( Sounds good, we should look to refactor this in the future.